src/Transport/HttpTransport.php

```diff
@@ -88,28 +89,32 @@ public function __destruct()
      */
     public function send(Event $event): ?string
     {
-        $projectId = $this->config->getProjectId();
-
-        if (null === $projectId) {
-            throw new MissingProjectIdCredentialException();
+        try {
```

Comment:
Instead of catching the entire code of the method we should just wrap the `wait` function call. The same should be done in the `cleanupPendingRequests` method. The strange thing is that when `false` is passed to the `wait` method no exceptions should be thrown but just returned (effectively doing the same as a `try`/`catch` block), so there is a bug somewhere in the vendor or the exception is not raised there (the other point where it could be raised is the `sendAsyncRequest` method, but since we saw that the request is not sent until `wait` is called I don't think it's the case)